core/src/integrationTest/java/org/lflang/tests/runtime/CPatmosTest.java

```diff
@@ -40,11 +40,24 @@ public CPatmosTest() {
   }
 
   @Test
-  public void buildPatmosBasicTestsUnthreaded() {
+  public void buildPatmosUnthreadedTests() {
+    Assumptions.assumeTrue(isLinux(), "Patmos tests only run on Linux");
+    super.runTestsFor(
+        List.of(Target.C),
+        Message.DESC_PATMOS,
+        TestCategory.PATMOS::equals,
+        Transformers::noChanges,
+        Configurators::makePatmosCompatibleUnthreaded,
+        TestLevel.BUILD,
```

Comment:
Consider running the tests instead of just building them and testing them later.